content/fr/docs/concepts/services-networking/endpoint-slices.md

```diff
@@ -0,0 +1,111 @@
+---
+reviewers: 
+title: EndpointSlices
+feature:
+  title: EndpointSlices
+  description: >
+    Suivi évolutif des points de terminaison réseau dans un cluster Kubernetes.
+
+content_template: templates/concept
+weight: 10
+---
+
+
+{{% capture overview %}}
+
+{{< feature-state for_k8s_version="v1.17" state="beta" >}}
+
+_EndpointSlices_ offrent une simple methode pour suivre les endpoints d'un réseau au sein d'un cluster de Kubernetes. Ils offrent une alternative plus evolutive et extensible aux Endpoints.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Resource pour EndpointSlice {#endpointslice-resource}
+
+Dans Kubernetes, un EndpointSlice contient des reférences à un ensemble de reseau 
+endpoints. Le controleur d'EndpointSlice crée automatiquement des EndpointSlices pour un Kubernetes Service quand un {{< glossary_tooltip text="selecteur" term_id="selecteur" >}} est spécifié. Ces EnpointSlices vont inclure des references a n'importe quelle Pods qui correspond aux selecteur de Service. EndpointSlices groupent ensemble les endpoints d'un reseau par combinaisons uniques de Services et de Ports.
```

Comment:
@Luciowl the build failure seems due to this error `9:32:40 AM: ERROR 2020/03/02 14:32:40 [fr] "docs/concepts/services-networking/endpoint-slices.md": "selecteur" is not a valid glossary term_id, see ./docs/reference/glossary/* for a full list`